docs/src/main/asciidoc/reactive-sql-clients.adoc

```diff
@@ -184,14 +184,21 @@ It returns a `Uni` and thus can be composed to execute queries sequentially:
 
 [source,java]
 ----
- client.query("DROP TABLE IF EXISTS fruits")
-    .flatMap(r -> client.query("CREATE TABLE fruits (id SERIAL PRIMARY KEY, name TEXT NOT NULL)"))
-    .flatMap(r -> client.query("INSERT INTO fruits (name) VALUES ('Orange')"))
-    .flatMap(r -> client.query("INSERT INTO fruits (name) VALUES ('Pear')"))
-    .flatMap(r -> client.query("INSERT INTO fruits (name) VALUES ('Apple')"))
+ client.query("DROP TABLE IF EXISTS fruits").execute()
+    .flatMap(r -> client.query("CREATE TABLE fruits (id SERIAL PRIMARY KEY, name TEXT NOT NULL)").execute())
+    .flatMap(r -> client.query("INSERT INTO fruits (name) VALUES ('Orange')").execute())
+    .flatMap(r -> client.query("INSERT INTO fruits (name) VALUES ('Pear')").execute())
+    .flatMap(r -> client.query("INSERT INTO fruits (name) VALUES ('Apple')").execute())
     .await().indefinitely();
 ----
 
+[IMPORTANT]
+.Breaking Change
+====
+Vert.x 3.9 introduces a breaking change regarding the `query` methods.
```

Comment:
We also need the Quarkus version here. People won't know in which version we introduced the change.

Reply:
Yes, but I didn't know the version until today.

Reply:
Note that I need to add a few lines to the breaking changes document

Reply:
Done.